src/gtfs/read.rs

```diff
@@ -607,7 +610,14 @@ where
             let mut rdr = csv::Reader::from_reader(reader);
             let mut transfers = vec![];
             for transfer in rdr.deserialize() {
-                let transfer: Transfer = transfer.with_context(ctx_from_path!(path))?;
+                let transfer: Transfer = match transfer {
```

Comment:
you can use skip_fail!
```rust
let transfer: Transfer = skip_fail!(transfer.with_context(ctx_from_path!(path)));
```

Reply:
skip_fail! handle "with_context" error and not deserialize error

Reply:
skip_fail does exactly what you do with match + warn!